code/__HELPERS/text.dm

```diff
@@ -729,3 +729,8 @@
 		return "#e67e22" // Patreon orange
 	return null
 
+
+// Removes HTML tags, preserving text
+/proc/strip_html_tags(the_text)
+	var/static/regex/html_replacer = regex("<\[^>]*>", "g")
```

Comment:
Could you give an example of what this is trying to match? Nothing I try triggers a match.

Reply:
![image](https://user-images.githubusercontent.com/25063394/205141070-c16b7e71-4a9c-4868-9c5b-9a9f482e388d.png)